src/main/java/com/n1analytics/paillier/EncryptedNumber.java

```diff
@@ -18,73 +18,7 @@
 import java.math.BigInteger;
 
 /**
- * Immutable class representing encrypted number and arithmetic operations that can be computed on the encrypted number.
```

Comment:
This change seems to remove lots of clear high level documentation about an EncryptedNumber.